libraries/customisations/src/main/kotlin/com/bumble/appyx/utils/customisations/LazyMutableNodeCustomisationDirectory.kt

```diff
@@ -0,0 +1,10 @@
+package com.bumble.appyx.utils.customisations
+
+import kotlin.reflect.KClass
+
+interface LazyMutableNodeCustomisationDirectory: MutableNodeCustomisationDirectory {
```

Comment:
Sure thing. I was thinking about you specifically when I avoided breaking the API compatibility haha.

Reply:
should we release 1.2 in this case?

Reply:
Hmm. If you're happy to release 1.2 soon, otherwise I will just leave it as is, and add a comment saying to remove it in 1.2?

I would prefer to release this reasonably soon if possible

Reply:
if we extend `MutableNodeCustomisationDirectory` we'll break source compatibility. Yes, most likely nobody implemented their own `MutableNodeCustomisationDirectory`  but nevertheless it's source incompatible change. Maybe we can keep it as `LazyMutableNodeCustomisationDirectory` with `Deprecated("Will be merged into MutableNodeCustomisationDirectory in 1.2")` ?

Reply:
yeah, I will add the deprecated annotation